propagation/upscaling.md

```diff
@@ -5,4 +5,41 @@ over larger geographic areas. The service assumes that the algorithm is already
 at scale.
 
 The service primarily provides self-service tools that project teams or end-users can utilize to run an algorithm at scale, using
-the processing capabilities of the platforms that are compatible with the service.
+the processing capabilities of the platforms that are compatible with the service.These tools currently include:
+
+* An **openEO batch job manager**, which supports large-scale processing tasks by dividing them into separate batch jobs that are automatically monitored.
+
+:::{.callout-warning}
+# Work in progress
+
+A prototype of this specific APEx service is expected to be available from November 2024 onwards. 
+:::
+
+## Upscaling support
+
+As upscaling to large areas can be challenging, APEx offers the possibility to request time-limited support for your 
+ESA project. Based on your needs, APEx can provide:
+
+- Basic training in the use of the upscaling service.
+- Definition of an appropriate upscaling strategy, tailored to your needs.
+- Selection of the most appropriate platform for your upscaling task.
+
+In order to make use of the upscaling service, please be aware that careful planning is needed:
+ 
+ - Your service needs to be onboarded in APEx before upscaling can start. Please refer to our [Algorithm Hosting](./hosting.md) service for more information. 
```

Comment:
Would this be a "hard" requirement? Let's say the algorithm is still under development and they would like to see how it compares to others. Would those algorithms be onboarded first, but hidden in the catalogue? An alternative option could be that we say that the algorithm needs to be in line with the interoperability and compliance guidelines to ensure we get a good base algorithm to work with. What do you think?